packages/grid/src/elements/pane/Pane.tsx

```diff
@@ -16,17 +18,33 @@ const PaneComponent = forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>
   ({ children, ...props }, ref) => {
     const [paneId, setPaneId] = useState<string>();
 
+    const observerRef = useRef<HTMLDivElement>(null);
+    const { width = 0, height = 0 } = useResizeObserver<HTMLDivElement>({ ref: observerRef });
+
+    const isVisible = useMemo(() => {
+      if (observerRef.current) {
+        if (width < 1) {
+          return false;
+        } else if (height < 1) {
+          return false;
+        }
+      }
+
+      return true;
```

Comment:
```suggestion
      return observerRef.current ? width >= 0 && height >= 0 : true;
```

might be nice to compact this a bit 🤷